feg/protos/basic_acct.proto

```diff
@@ -49,16 +49,12 @@ message AcctSession {
         bytes  hardware_addr = 3;
         string name = 4;
     }
-    // ID of the user network (MNO, Identity Provider, etc.)
-    string user_network_id = 5;
     // unique session ID
-    string session_id = 6;
-    // ID of the service provider network
-    string serving_network_id = 7;
+    string session_id = 5;
     // ID of the service provider gateway (AP, AGW, enodeb, etc.)
-    string serving_gateway_id = 8;
+    string serving_apn = 6;
```

Comment:
If serving_apn is a rename of the existing session_id field, then this update is "safe". But if it's a new field entirely it should avoid re-use of the existing tag (6) and should instead use a new and not-previously-allocated tag.  This is best practice for backwards compatibility, even if you believe or know that in this case all producers and consumers of the FEG API will be synchronized (not sure if that is true).

Reply:
This is a brand new proto, there is no service using it yet. All changes are safe for now.